go/mysql/binlog_event_mysql56.go

```diff
@@ -89,8 +89,8 @@ func (ev mysql56BinlogEvent) StripChecksum(f BinlogFormat) (BinlogEvent, []byte,
 		// Checksum is the last 4 bytes of the event buffer.
 		data := ev.Bytes()
 		length := len(data)
-		checksum := data[length-4:]
-		data = data[:length-4]
+		checksum := data[length-binlogChecksumLen:]
+		data = data[:length-binlogChecksumLen]
```

Comment:
again, we must not assume `CRC32` here.

Reply:
(I realize it's not your PR that makes the assumption, but we should fix this assumptions while we're here)